wiki/People/Performance_Points_Committee/en.md

```diff
@@ -7,36 +7,81 @@ tags:
 
 The **Performance Points Committee** is a panel of well-versed players and developers who facilitate and approve updates to the [performance points](/wiki/Performance_points) and [star rating](/wiki/Beatmap/Star_rating) algorithms. Their goal is to help [osu! developers](/wiki/People/Developers) push changes faster and with more community coordination.
 
-The osu! panel was first announced to the public through the [November 2021 *Performance Points & Star Rating Updates* news post](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates) in response to a [community survey](https://osu.ppy.sh/home/news/2021-08-17-pp-sr-survey) held earlier that year.
+The Performance Points Committee's responsibilities include:
 
-The osu!mania panel was formed 22 April 2022 in response to [another community survey regarding performance points and star rating in osu!taiko and osu!mania](https://osu.ppy.sh/home/news/2022-01-14-taiko-mania-pp-sr-survey) held earlier that year, as well as in anticipation of a surge in active development effort, starting with [a change in difficulty calculation for hold note releases in osu!mania](https://github.com/ppy/osu/pull/17913).
+- Organizing development efforts and helping contributors
+- Reviewing and developing changes in difficulty calculation algorithms
+- Deciding which changes will be deployed and used in the game
+- Assisting with writing news posts for newly deployed changes
 
-The osu!taiko panel was formed on 1 May 2022 in response to the same survey as above, with [major changes to the star rating algorithm](https://docs.google.com/document/d/1Z5GC4DMqOVzeIERMSK3qpQaqjq-sVnhbuoxAwy9qxDs/edit).
+## Workflow
+
+Whenever a new difficulty calculation proposal is created on [GitHub](https://github.com/ppy/osu), the Performance Points Committee relevant to the ruleset it's for will be tagged with a review request. After that members of the committee must review and test the proposal and write a review on its GitHub page. When there are enough committee approvals the change is considered ready for deployment and is passed to osu! developers for final code inspection before merging into the game.
+
+When there are enough merged proposals the osu! developers will start preparing for a performance points deployment which will recalculate all existing scores using the new algorithms. Performance Points Committees are tasked with assisting with writing a news post about the upcoming changes.
```

Comment:
```suggestion
The osu! developers prepare merged proposals to be deployed in batches. A deployment recalculates star ratings for all beatmaps and performance points for all scores and users. The Performance Points Committee helps write the news post that accompanies each deployment.
```
maybe? remove "enough" here again bc afaik that's not even a thing, standalone description of what a "deployment" is, some other reword